test/attestations.test.ts

```diff
@@ -0,0 +1,96 @@
+import {ethers} from "@nomiclabs/buidler";
+import {assert} from "chai";
+import {submit} from "./support";
+import {AttestationRegistryFactory} from "../ethers-contracts";
+
+describe("Attestations", function () {
+  it("should allow attestations to be made and revoked", async function () {
+    const [signer] = await ethers.getSigners();
+    const address = await signer.getAddress();
+    const attestationRegistry = await new AttestationRegistryFactory(
+      signer
+    ).deploy();
+    await attestationRegistry.deployed();
+
+    const id = ethers.utils.randomBytes(32);
+    const pk = ethers.utils.randomBytes(32);
+    const sig = [
```

Comment:
```suggestion
    const sig = new Array(64).fill([0])
```

Reply:
Was hoping someone would help me out here, I spent 30 mins trying to unify the types.